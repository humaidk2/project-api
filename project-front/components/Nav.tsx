import React from "react"
const Nav = ({ username }) => {
    return (
        <nav className="navigation">
            <div>{`Hi, Welcome to ${username}'s page`}</div>
            <style jsx>
                {`
                    .navigation {
                        height: 60px;
                        z-index: 1;
                        top: 0;
                        left: 0;
                        width: 100%;
                        color: #f9f7f7;
                        background: #112d4e;
                        border-bottom: 0.2px solid rgba(17, 45, 78, 0.2);
                        box-shadow: 0 20px 20px rgba(17, 45, 78, 0.1);
                    }
                `}
            </style>
        </nav>
    )
}
export default Nav
