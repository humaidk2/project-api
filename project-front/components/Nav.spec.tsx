import * as React from "react"
import { shallow } from "enzyme"
import Nav from "./Nav"

describe("Nav component", () => {
    describe("Nav", () => {
        it("should render without throwing an error", function () {
            const wrap = shallow(<Nav username="test" />)
            expect(wrap).toMatchSnapshot()
        })
    })
})
