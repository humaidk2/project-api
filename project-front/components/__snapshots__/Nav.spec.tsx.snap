// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component Nav should render without throwing an error 1`] = `
<nav
  className="navigation"
>
  <div>
    Hi, Welcome to test's page
  </div>
  <style
    jsx={true}
  >
    
                    .navigation {
                        height: 60px;
                        z-index: 1;
                        top: 0;
                        left: 0;
                        width: 100%;
                        color: #f9f7f7;
                        background: #112d4e;
                        border-bottom: 0.2px solid rgba(17, 45, 78, 0.2);
                        box-shadow: 0 20px 20px rgba(17, 45, 78, 0.1);
                        display: flex;
                        justify-content: flex-end;
                        align-items: center;
                        padding: 5px;
                        padding-right: 10px;
                    }
                    .navigation div {
                        margin: 5px;
                    }
                
  </style>
</nav>
`;
